packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -93,6 +90,9 @@ const SignUpPasswordView = FormView.extend({
         AuthErrors.toError('PASSWORDS_DO_NOT_MATCH'),
         false
       );
+    } else {
+      // Called to clear validation tooltips.
+      this.$(VPASSWORD_INPUT_SELECTOR).change();
```

Comment:
This fixes other two issues. If the tooltip is on an element whose value does not change, the the validation rules to remove it were not triggered. Simple fix was to fake an element change.